apps/catalogue/src/components/CohortCards.vue

```diff
@@ -0,0 +1,135 @@
+<template>
```

Comment:
implement as display component to allow for consistency and reuse

Reply:
wontfix for now: this is still using the TableMixin. Want to move to client at some point and then this logic can move to the view.